api/src/main/java/uk/gov/dhsc/htbhf/eligibility/factory/EligibilityResponseFactory.java

```diff
@@ -49,10 +54,27 @@ public EligibilityResponse createEligibilityResponse(DWPEligibilityResponse dwpR
 
         builder.numberOfChildrenUnderOne(nullSafeMax(dwpResponse.getNumberOfChildrenUnderOne(), hmrcResponse.getNumberOfChildrenUnderOne()));
         builder.numberOfChildrenUnderFour(nullSafeMax(dwpResponse.getNumberOfChildrenUnderFour(), hmrcResponse.getNumberOfChildrenUnderFour()));
+        builder.children(createChildren(dwpResponse, hmrcResponse));
 
         return builder.build();
     }
 
+    private List<ChildDTO> createChildren(DWPEligibilityResponse dwpResponse, HMRCEligibilityResponse hmrcResponse) {
+        List<ChildDTO> dwpChildren = getEmptyListIfNull(dwpResponse.getChildren());
+        List<ChildDTO> hmrcChildren = getEmptyListIfNull(hmrcResponse.getChildren());
+
+        if (dwpChildren.size() >= hmrcChildren.size()) {
```

Comment:
I wasn't aware of this choice, is this documented somewhere, seems an important thing to do with something like this?

Reply:
Updating the top level javadoc explaining the rules regarding children.